src/typeahead/highlight.ts

```diff
@@ -0,0 +1,43 @@
+import {Component, Input, OnChanges, ChangeDetectionStrategy} from '@angular/core';
+import {regExpEscape} from '../util/util';
+
+function toDefinedString(val: any) {
+  return (val !== undefined && val !== null) ? `${val}` : '';
+}
+
+@Component({
+  selector: 'ngb-highlight',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `<template ngFor [ngForOf]="_parts" let-part let-isOdd="odd">` +
```

Comment:
This "certain way" you meantion is having to contact them instead of putting them in a multiline format with the backsticks?